### PR TITLE
AJ-762 address set-output and save-state deprecation warnings

### PR DESCRIPTION
**Pull request by calypsomatic**
Ticket: <Link to Jira ticket>
<Put notes here to help reviewer understand this PR>

---

**PR checklist**

- [ ] Include the JIRA issue number in the PR description and title
- [ ] Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] If you changed anything in `model/`, then you should [publish a new official `rawls-model`](https://github.com/broadinstitute/rawls/blob/develop/README.md#publish-rawls-model) and update `rawls-model` in [Orchestration's dependencies](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/project/Dependencies.scala).
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green, including CI tests
- [ ] **Squash commits and merge** to develop (branches are automatically deleted after merging)
- [ ] Inform other teams of any substantial changes via Slack and/or email

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
 
       # coursier cache action caches both coursier and sbt caches
       - name: coursier-cache-action
-        uses: coursier/cache-action@v5
+        uses: coursier/cache-action@v6
```
Comment (aaronkanzer) [inline .github/workflows/test.yml:21]: Seems there are `set-output` references in rawls-build-tag-publish-and-run-tests.yaml too

Comment (davidangb) [inline .github/workflows/test.yml:21]: There is a `coursier/cache-action@v5` in format.yaml that needs updating: https://github.com/broadinstitute/rawls/blob/af2435c0e833d70a346d65891c7f8c511f3eadb5/.github/workflows/format.yaml#L20

**Review by aaronkanzer — APPROVED** [review]

**Review by davidangb — APPROVED** [review]
:thankyou:

File: .github/workflows/rawls-build-tag-publish-and-run-tests.yaml [hunk 2]
```diff
@@ -43,8 +43,8 @@ jobs:
             exit 1
           fi
 
-          echo ::set-output name=ref::$GITHUB_REF
```
Comment (aaronkanzer) [inline .github/workflows/rawls-build-tag-publish-and-run-tests.yaml:46]: Flag for Yulia's PR

**Pull request merged**